docs/Events/bpmn-events/import-render-complete.md

```diff
@@ -4,9 +4,42 @@ title: import.render.complete
 nav_order: 7
 description: import.render.complete event
 permalink: /events/bpmn-events/import-render-complete
-last_modified_date: 2020-04-27T17:54:08+0000
+last_modified_date: 2020-09-21T12:00:00+0000
 parent: "BPMN Events"
 grand_parent: "Events"
 ---
 
 # import.render.complete
+
+This event is fired by [importXML API](/general/api-reference/bpmn-js/importXML) with data containing `errors` and `warnings` after the graphical import ends. This allows users to listen to whether the rendering of a diagram was successful or not.
+
+## Example Usage
+
+```javascript
+// modeler is an instance of BpmnJS Modeler.
+
+modeler.on('import.render.complete', (data) => {
+
+  const { errors, warnings } = data; 
+
+  // check for errors while importing the diagram
+  if(errors && errors.length) {
+    console.log('Errors while importing the diagram raised:', errors);
+  }
+
+  // check for warnings while importing the diagram
+  if(warnings && warnings.length) {
+    console.log('Warnings while importing the diagram raised:', warnings);
+  }
+});
+```
+
+## Data
+
+The data fired with this event contains these fields:
+
+`errors`: _Array\<[String](#)\>_ <br>
```

Comment:
`import.render.complete` also contains an `error` field which is an instance of `Error` rathar than `errors` which is an array of string: https://github.com/bpmn-io/bpmn-js/blob/36e4f6113c5fc1e826894244c6da39636b75861c/lib/import/Importer.js#L82

Reply:
Don't know why I thought the given error would be a list instead of a single object...